fabric-convention-tags-v2/src/main/java/net/fabricmc/fabric/api/tag/convention/v2/TagUtil.java

```diff
@@ -81,4 +82,29 @@ public static <T> boolean isIn(@Nullable DynamicRegistryManager registryManager,
 
 		return false;
 	}
+
+	/**
+	 * Use this to get a TagKey's translation key safely on any side.
+	 * @return the translation key for a TagKey.
+	 */
+	public static String getTagTranslationKey(TagKey<?> tagKey) {
+		StringBuilder stringBuilder = new StringBuilder();
+		stringBuilder.append("tag.");
+
+		Identifier registryIdentifier = tagKey.registry().getValue();
+		Identifier tagIdentifier = tagKey.id();
+
+		if (registryIdentifier.getNamespace().equals("minecraft")) {
```

Comment:
Is this logic inverted?

```suggestion
		if (!registryIdentifier.getNamespace().equals("minecraft")) {
```